linux/LinuxProcessList.c

```diff
@@ -607,9 +612,14 @@ static bool LinuxProcessList_updateUser(ProcessList* processList, Process* proce
    return true;
 }
 
-static void LinuxProcessList_readIoFile(LinuxProcess* process, openat_arg_t procFd, unsigned long long realtimeMs) {
+static void LinuxProcessList_readIoFile(LinuxProcess* process, openat_arg_t procFd, bool scanMainThread, unsigned long long realtimeMs) {
+   Process *proc = &process->super;
```

Comment:
Naming convention: `LinuxProcess *lp` vs. `Process *process`.

Reply:
You want this?

```
diff --git a/linux/LinuxProcessList.c b/linux/LinuxProcessList.c
index b94c24a1..351c545f 100644
--- a/linux/LinuxProcessList.c
+++ b/linux/LinuxProcessList.c
@@ -612,8 +612,8 @@ static bool LinuxProcessList_updateUser(ProcessList* processList, Process* proce
    return true;
 }
 
-static void LinuxProcessList_readIoFile(LinuxProcess* process, openat_arg_t procFd, bool scanMainThread, unsigned long long realtimeMs) {
-   Process *proc = &process->super;
+static void LinuxProcessList_readIoFile(LinuxProcess* lp, openat_arg_t procFd, bool scanMainThread, unsigned long long realtimeMs) {
+   Process *proc = &lp->super;
    char path[20] = "io";
    char buffer[1024];
    if (scanMainThread) {
@@ -621,16 +621,16 @@ static void LinuxProcessList_readIoFile(LinuxProcess* process, openat_arg_t proc
    }
    ssize_t r = xReadfileat(procFd, path, buffer, sizeof(buffer));
    if (r < 0) {
-      process->io_rate_read_bps = NAN;
-      process->io_rate_write_bps = NAN;
-      process->io_rchar = ULLONG_MAX;
-      process->io_wchar = ULLONG_MAX;
-      process->io_syscr = ULLONG_MAX;
-      process->io_syscw = ULLONG_MAX;
-      process->io_read_bytes = ULLONG_MAX;
-      process->io_write_bytes = ULLONG_MAX;
-      process->io_cancelled_write_bytes = ULLONG_MAX;
-      process->io_last_scan_time_ms = realtimeMs;
+      lp->io_rate_read_bps = NAN;
+      lp->io_rate_write_bps = NAN;
+      lp->io_rchar = ULLONG_MAX;
+      lp->io_wchar = ULLONG_MAX;
+      lp->io_syscr = ULLONG_MAX;
+      lp->io_syscw = ULLONG_MAX;
+      lp->io_read_bytes = ULLONG_MAX;
+      lp->io_write_bytes = ULLONG_MAX;
+      lp->io_cancelled_write_bytes = ULLONG_MAX;
+      lp->io_last_scan_time_ms = realtimeMs;
       return;
    }
 

```

Reply:
Even though this is somewhat more extensive than the initial patch, from a naming convention consistency PoV this would be better. You may put this as a separate commit as this is mostly a stylistic change.

P.S.: You can put a programming language after the 3 backticks in markdown to use syntax highlighting. In case you wondered what that edit was about.

Reply:
>Even though this is somewhat more extensive than the initial patch, from a naming convention consistency PoV this would be better. You may put this as a separate commit as this is mostly a stylistic change.

Ok, then, possible, on separate pull request, since current LinuxProcessList_readIoFile() @process argument name is not a blame of current pull request ;)

Reply:
You can split this, but as offered, you may include it here as a second commit too.